CHANGELOG.rst

```diff
@@ -6,6 +6,34 @@ complete overview of changes, please refer to the git log.
 The format is based on `Keep a Changelog`_,
 but most notably, without sectioning changes into type-of-change.
 
+0.3.0_ - 2020.03.08
+-------------------
+* Added an initial pass at a Log Information Standard 79 (LIS79) reader. Like
+  the DLIS reader, the new LIS reader is mainly implemented in C/C++ with
+  python bindings on top. The reader is not feature complete at this point. But
+  it can read most curves, with a few exceptions (see the docs of
+  dlisio.lis.curves). Basic metadata such as LIS Header/trailer Records (RHLR,
+  RTLR, THLR, TTLR, FHLR, FTLR) can also be read. Support for more complex LIS
+  Records such Information Records is not yet added. The LIS reader
+  resides in the python submodule ``dlisio.lis``.
+* The python module is restructured to accommodate the new LIS reader. Most
+  notably, all DLIS related functionality is moved to the submodule
+  ``dlisio.dlis``. I.e.  this release breaks the main entry point of dlisio
+  ``dlisio.load`` which from this release and onwards is moved to
+  ``dlisio.dlis.load``. For a full overview of the restructuring see
+  commit ``#736d545``.
+* The documentation on readthedocs_ has been given an overhaul to fit the new
+  module structure and LIS documentation is added.
+* Added support for DLIS NO-FORM objects
+* Better debug information for broken DLIS files.
+* Better error message when trying to open non-existing DLIS files.
```

Comment:
I think we had it before? Recent thing in this area is an error message for opening directories (which I am often guilty of...)